src/compiler/transformers/decorators-to-static/watch-decorator.ts

```diff
@@ -1,20 +1,12 @@
-import { augmentDiagnosticWithNode, buildError, buildWarn, flatOne } from '@utils';
+import { flatOne } from '@utils';
 import ts from 'typescript';
 
 import type * as d from '../../../declarations';
 import { convertValueToLiteral, createStaticGetter, retrieveTsDecorators } from '../transform-utils';
 import { getDeclarationParameters, isDecoratorNamed } from './decorator-utils';
 
-export const watchDecoratorsToStatic = (
-  config: d.Config,
-  diagnostics: d.Diagnostic[],
-  decoratedProps: ts.ClassElement[],
-  watchable: Set<string>,
-  newMembers: ts.ClassElement[],
-) => {
-  const watchers = decoratedProps
-    .filter(ts.isMethodDeclaration)
-    .map((method) => parseWatchDecorator(config, diagnostics, watchable, method));
+export const watchDecoratorsToStatic = (decoratedProps: ts.ClassElement[], newMembers: ts.ClassElement[]) => {
+  const watchers = decoratedProps.filter(ts.isMethodDeclaration).map((method) => parseWatchDecorator(method));
```

Comment:
super nit: this could be slightly more concise

```suggestion
  const watchers = decoratedProps.filter(ts.isMethodDeclaration).map(parseWatchDecorator);
```